people/zuzana.json

```diff
@@ -0,0 +1,22 @@
+{
+
+  "key": "zuzana",
+
+  "name": "BcA. Zuzana Červínová",
+
+  "nick": "Zuzana",
+
+  "email": "zuzana.cervinova@mild.blue",
+
+  "title_cs": "JUNIOR FRONTEND DEVELOPER",
+
+  "title_en": "JUNIOR FRONTEND DEVELOPER",
+
+  "short_desc_en": "Zuzana is a Bachelor of Arts and a graduate of the Czechitas Digital Academy: Web. The skills she gained during the course, as well as her artistic sense, Zuzana now uses at Mild Blue, where she works as a junior frontend developer.",
+
+  "short_desc_de": "Zuzana ist Bachelor of Arts und Absolventin der Czechitas Digital Academy: Web. Ihre im Studium erworbenen Kenntnisse sowie ihr künstlerisches Gespür werden nun bei Mild Blue als Junior Frontend Developer eingesetzt und weiterentwickelt.
+  ",
```

Comment:
nemuzes mit uvozkovky na zacatku radku v jsonu. takhle je to nevalidni (i github ti to zabarvuje cervene  )